### ISSUE TITLE
Mangadex: Manga with no chapters should at least display cover and description or completely hidden

### ISSUE THREAD

**Issue by Luro02** [faldez/tanoshi-extensions#21]
When you click on a Manga that has no chapters, you will see something like this:

![image](https://user-images.githubusercontent.com/24826124/129882066-1df05994-76b5-4006-b749-0ecc305c42f4.png)

This suggests to the user that the extension is broken. When you look on mangadex, you see that the manga has no chapters, and that seems to be why nothing is displayed.

The logs seem to suggest this too:
```
tanoshi    | [2021-08-18T10:19:00Z DEBUG tanoshi_vm::extension_thread] call get_chapters("/manga/d8a959f7-648e-4c8d-8f23-f1f3f8e129f3") => (data:Some([]),error:None)
```

As a side-note: The button on the top left is no longer working, when you click on a manga with no chapters, a refresh of the page seems to solve this

**Comment by faldez** [comment]
ah I've never encountered manga with no chapter, what manga is this?

**Comment by Luro02** [comment]
This is happening for One Punch-Man

**Comment by faldez** [comment]
This is fixed ok tanoshi v0.25.4
